pom.xml

```diff
@@ -25,9 +25,9 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <parent>
-    <groupId>org.xwiki.commons</groupId>
-    <artifactId>xwiki-commons-pom</artifactId>
-    <version>5.2.4</version>
+    <groupId>org.xwiki.contrib</groupId>
```

Comment:
This is just the beginning. You should also:

* get rid of `<distributionManagement>` (it's already in the parent)
* get rid of `<issueManagement>` (kind of already in the parent)
* indicate the jira id trough properties and enable jira release (see https://github.com/xwiki-contrib/ldap/blob/master/pom.xml#L37)